sky/provision/azure/instance.py

```diff
@@ -274,6 +275,16 @@ def _create_instances(
         deployment_name=vm_name,
         parameters=parameters,
     ).wait()
+
+    if 'disk_performance_tier' in provider_config:
+        performance_tier = provider_config['disk_performance_tier']
+        disks = compute_client.disks.list_by_resource_group(resource_group)
+        for disk in disks:
+            name = disk.name
+            subprocess_utils.run_no_outputs(
+                f'az disk update -n {name} -g {resource_group} '
+                f'--set tier={performance_tier}')
```

Comment:
disk performance information feels more like a `node_config` instead of `provider_config`. Can we place it in node config instead?

Reply:
Moved. Thanks!